velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -871,7 +881,9 @@ class ParquetRowReader::Impl {
     firstRowOfRowGroup_.reserve(rowGroups_.size());
 
     ParquetData::FilterRowGroupsResult res;
-    columnReader_->filterRowGroups(0, ParquetStatsContext(), res);
+    ParquetStatsContext parquetStatsContext =
```

Comment:
You can store it as a field in the class so we don't recreate it for each row group

Reply:
done~